OLM_VERSION

```diff
@@ -1 +1 @@
-0.19.0
+0.22.0
```

Comment:
IMO, this file is tech debt and doesn't affect anything upstream outside of the related-related Makefile targets. Do we have any context for why we're bumping this version vs. working towards removing it entirely from the repository and deriving any versioning information from git commands?

Reply:
@timflannagan I just linked this PR in issue #2872 that should have the context. We need to have the v0.22.0 manifests to unblock our upstream consumers, till we figure out an alternative.

Reply:
Or do you mean it's sufficient to keep hardcoding the version in the target instead of have it fetch from the OLM_VERSION file till we figure out the alternative?

I'm fine with anything, just as long as we release the v0.22.0 manifests (and don't keep missing out on releasing manifests for the future releases)

Reply:
I've asked if the release artifacts can be used instead of using the charts

/hold 

till we hear back.

Reply:
@timflannagan looks like there are consumers who have use cases for the helm charts https://github.com/operator-framework/operator-lifecycle-manager/issues/2872#issuecomment-1276426894

I don't know enough about FluxCD to pose any argument for/against that, so I'm inclined to just release the v0.22.0 manifests (and keep releasing future manifests) in tree. 

/hold cancel